lib/ice_cube/integer_util.rb

```diff
@@ -0,0 +1,9 @@
+module IntegerUtil
+  def self.klass
+    if RUBY_VERSION.include?("2.4.")
```

Comment:
I think this probably should be something more concrete / robust - given versions after 2.4 will also have this change.  Helpful to look @ https://github.com/rails/rails/pull/25056